.github/workflows/macos.yaml

```diff
@@ -17,14 +17,26 @@ jobs:
         fetch-depth: "0"
     # Get tags not fetched by the checkout action, needed for auto-versioning.
     - run: "git fetch origin +refs/tags/*:refs/tags/*"
+
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
+
+    - name: Use pip cache
+      uses: actions/cache@v2
```

Comment:
A link to any docs for this action might be a nice addition here.

Reply:
Added a link to [relevant example](https://github.com/actions/cache/blob/master/examples.md#python---pip) in a38e077.